terraform/backend.tf

```diff
@@ -0,0 +1,80 @@
+resource "docker_container" "database" {
+  name  = "database"
+  image = var.database-image
```

Comment:
You should use a `docker_image` resource to pass the hash to the docker container